File: server/check_xlsx.py
import pandas as pd
try:
    df = pd.read_excel(r"C:\Users\NILE\Downloads\P.7.S-students-20260506.xlsx")
    print("Columns:", df.columns.tolist())
    print("First row:", df.iloc[0].to_dict())
except Exception as e:
    print("Error:", str(e))

File: server/check_xlsx.js
const xlsx = require('xlsx');
try {
    const workbook = xlsx.readFile('C:\\Users\\NILE\\Downloads\\P.7.S-students-20260506.xlsx');
    const sheet = workbook.Sheets['Students'] || workbook.Sheets[workbook.SheetNames[0]];
    const csv = xlsx.utils.sheet_to_csv(sheet);
    const header = csv.split('\n')[0];
    console.log('Full Headers:', header);
} catch (err) {
    console.log('Error:', err.message);
}
